docs/git-resolver.md

```diff
@@ -195,6 +191,113 @@ spec:
     value: Ranni
 ```
 
+### Specifying Configuration for Multiple Git Providers
+
+You can specify configurations for multiple providers and even multiple configurations for same provider to use in 
+different tekton resources. You need to pass the identifier mentioned in configmap as an extra param with key 
+`token-identifier`. If no identifier passed, `default` value will be considered. You can specify default configuration
+in configmap either by mentioning no identifier or by using default identifier `identifier.default`
+
+### Sample Configmap
+
+You can add multiple configuration to `git-resolver-configmap` like this. All keys mentioned above are supported.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: git-resolver-config
+  namespace: tekton-pipelines-resolvers
+  labels:
+    app.kubernetes.io/component: resolvers
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines
+data:
+  # configuration 1, default one to use if no identifier provided
+  fetch-timeout: "1m"
+  default-url: "https://github.com/tektoncd/catalog.git"
+  default-revision: "main"
+  scm-type: "github"
+  server-url: ""
+  api-token-secret-name: ""
+  api-token-secret-key: ""
+  api-token-secret-namespace: "default"
+  default-org: ""
+
+  # configuration 2, will be used if token-identifier params passed with value test1
+  identifier.test1.fetch-timeout: "5m"
```

Comment:
i think this looks good too. Idea to add `identifier` prefix to parse only the keys with this prefix for extra configs, and also it resonates with the extra param pass with key `token-identifier` but i am fine if we want to change it like that cc @vdemeester @chitrangpatel for your thoughts.

Reply:
we could have `config.test1.…` maybe ? I understand the "need" to have a prefix so that we can easily parse them and not have possible name conflicts (if someone wants to name its configuration "scm-type" for example 😂). 

But I would also be fine to go with `<config-name>.<config-key>: value` as I don't think there will be much more conflicts.

Reply:
Did like `<config-name>.<config-key>: value`